cinema/models.py

```diff
@@ -134,9 +134,8 @@ def save(
         )
 
     def __str__(self):
-        return (
-            f"{str(self.movie_session)} (row: {self.row}, seat: {self.seat})"
-        )
+        return f"{str(self.movie_session)} " \
```

Comment:
why did you change this?

Reply:
better to use ()

Reply:
Just to solve flake8 mistake. But, as I see, in this task it's not necessary 😟